### ISSUE TITLE
2019-05-17 Play 2.7.x nightly failure

### ISSUE THREAD

**Issue by dwijnand** [playframework/play-meta#76]
https://travis-ci.com/playframework/playframework/builds/112165663

In the validation stage (1st stage) the validate docs and validate microbenchmarks failed with

```
com.typesafe.akka#akka-http-core_2.12;10.1.8+22-be8241d5: not found
```

**Comment by dwijnand** [comment]
That jar seems to be there now.  Maybe a race condition on Akka HTTP's nightly publish?  Or an even-harder-to-hit race condition on an Akka HTTP's publish on (PR or master) commit push??  Restarted the job to check.

**Comment by dwijnand** [comment]
Fixed in playframwork/playframwork#9322 and playframwork/playframwork#9323.
